updating/index.adoc

```diff
@@ -8,29 +8,43 @@ toc::[]
 
 You can update an {product-title} 4 cluster with a single operation by using the web console or the OpenShift CLI (`oc`).
 
+ifndef::openshift-origin[]
 [id="updating-clusters-overview-understanding-container-platform-updates"]
 == Understanding OpenShift Container Platform updates
 xref:../updating/understanding-openshift-updates.adoc#update-service-about_understanding-openshift-updates[About the OpenShift Update Service]: For clusters with internet access, Red Hat provides over-the-air updates by using an {product-title} update service as a hosted service located behind public APIs.
+endif::openshift-origin[]
 
 [id="updating-clusters-overview-upgrade-channels-and-releases"]
 == Understanding upgrade channels and releases
+ifndef::openshift-origin[]
 xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrade channels and releases]: With upgrade channels, you can choose an upgrade strategy. Upgrade channels are specific to a minor version of {product-title}. Upgrade channels only control release selection and do not impact the version of the cluster that you install. The `openshift-install` binary file for a specific version of the {product-title} always installs that minor version. For more information, see the following:
 
 * xref:../updating/understanding-upgrade-channels-release.adoc#upgrade-version-paths_understanding-upgrade-channels-releases[Upgrading version paths]
 * xref:../updating/understanding-upgrade-channels-release.adoc#fast-stable-channel-strategies_understanding-upgrade-channels-releases[Understanding fast and stable channel use and strategies]
 * xref:../updating/understanding-upgrade-channels-release.adoc#restricted-network-clusters_understanding-upgrade-channels-releases[Understanding restricted network clusters]
 * xref:../updating/understanding-upgrade-channels-release.adoc#switching-between-channels_understanding-upgrade-channels-releases[Switching between channels]
 * xref:../updating/understanding-upgrade-channels-release.adoc#conditional-updates-overview_understanding-upgrade-channels-releases[Understanding conditional updates]
+endif::openshift-origin[]
+ifdef::openshift-origin[]
+With upgrade channels, you can choose an upgrade strategy. Upgrade channels are specific to a minor version of {product-title}. Upgrade channels only control release selection and do not impact the version of the cluster that you install. The `openshift-install` binary file for a specific version of the {product-title} always installs that minor version.
+
```

Comment:
Peer review: I did not change this paragraph from [Line 20](https://github.com/openshift/openshift-docs/pull/58413/files#diff-24163360618968f63d32275a3f76bdddaf7f4d60ac9ef56a8a51433ea9b8cd87R20), except to remove the link.